homeworks/src/vkochergin/module1/example/Homework2/HackerRankDay1.java

```diff
@@ -0,0 +1,40 @@
+package vkochergin.module1.example.Homework2;
+
+import java.util.Scanner;
+
+/**
+ * Created by Volodymyr Kochergin on 11.04.2017.
+ */
+public class HackerRankDay1 {
+    public static void main(String[] args) {
+        int i = 4;
+        double d = 4.0;
+        String s = "HackerRank ";
+
+        Scanner scan = new Scanner(System.in);
+
+                /* Declare second integer, double, and String variables. */
+        int secInt;
+        double secDouble;
+        String secString;
+
+        /* Read and save an integer, double, and String to your variables.*/
+        // Note: If you have trouble reading the entire String, please go back and review the Tutorial closely.
+        secInt = scan.nextInt();
+        secDouble = scan.nextDouble();
+        secString = scan.next();

```

Comment:
Why do you need next() to be called two times? Maybe better use nextLine() method